File: core/src/com/mygdx/game/model/MachineGun.kt
package com.mygdx.game.model

import com.mygdx.game.settings.MACHINE_FUN_RELOAD_TIME
import com.mygdx.game.settings.MACHINE_GUN_BULLETS_IN_CHAMBER

class MachineGun : Weapon(MACHINE_FUN_RELOAD_TIME, MACHINE_GUN_BULLETS_IN_CHAMBER, ProjectileType.MACHINE_GUN)

File: core/src/frontendServer/server/models/MachineGun.js
const constants = require('../settings/constants');
const Weapon = require('./Weapon');
const ProjectileType = require('./ProjectileType');

class MachineGun extends Weapon {
    constructor() {
        super(constants.MACHINE_GUN_RELOAD_TIME, constants.MACHINE_GUN_BULLETS_IN_CHAMBER,
            constants.MACHINE_GUN_MAGAZINE_RELOAD_TIME, ProjectileType.MACHINE_GUN)
    }
}

module.exports = MachineGun;